### DISCUSSION TITLE
available_quantity não atualiza em alguns anúncios

### DISCUSSION THREAD

**Discussion by MikeGBarreto** [rodurma/forum-meli#12] [Anúncios (items)]
Fala pessoal!
Alguém ai já pegou esse problema...

Consigo atualizar o estoque de quase todos os anúncios via API, más alguns anúncios retorna o seguinte erro:
**Product stock Fulfillment. Unable to update available_quantity**
MAS ESSES ANUNCIOS NÃO SÃO FULFILLMENT!

Quando faço um GET para os anúncios que apresentam esse problema, notei que todos tem o parâmetro **inventory_id** com um ID válido, e o parâmetro **shipping.logistic_type** está preenchido com o valor **cross_docking**.

Esse comportamento me parece estranho. Tendo em vista que na documentação diz que o inventory_id serve para obter os dados do item com estoque do tipo fulfillment. https://developers.mercadolivre.com.br/pt_br/estoque-de-fulfillment.

Já nos anúncios que consigo obter sucesso na atualização do estoque o parâmetro **inventory_id** está vazio e o parâmetro **shipping.logistic_type** continua com valor **cross_docking**. Creio que esse deveria ser o retorno para os anúncios que estão retornando erro.

Se eu atualizar o estoque via painel do ML desses itens com erro, nenhum erro é retornado. Ou seja, de fato não é um anuncio fulfillment.

Se eu criar um novo anuncio a partir do anuncio que apresenta esse erro, adivinha... Consigo atualizar o estoque do novo anuncio  normalmente via API.

Alguém ai pode me dar uma luz?

**Comment by JuanCalavera** [comment]
No seu caso eles podem ter ainda reservado no estoque deles, mas esta "reserva" se encontra perdida ou danificada, o certo é criar uma duplicata do anúncio pois o status vai direto para drop_off.

**Reply by MikeGBarreto** [reply]
Fala Juan, blz! 
Sim. A duplicata funciona perfeitamente. O problema é que ao criar um novo anuncio não dá pra aproveitar o histórico de vendas do produto.
